.github/workflows/tests.yml

```
@@ -836,6 +838,10 @@ jobs:
               prefix = "${{ github.ref_name }}"[:-1]  # without x suffix
               if not last_release.startswith(prefix):
                   envs["VERSION_OVERRIDE"] = "${{ github.ref_name }}"
+              else:
+                last_tag = check_output(["git", "describe", "--tag", "--abbrev=0"], text=True).strip()
+                num_commits = check_output(["git", "rev-list", "--count", "HEAD", f"{last_tag}..HEAD"], text=True).strip()
```

`git describe --tag` already includes the number of commits since the last tag:

```suggestion
                if "-" in last_release:
                    last_tag, num_commits, _ = last_release.split("-")
                else:
                    last_tag, num_commits = last_release, "0"
```